engine/menu.asm

```diff
@@ -730,15 +730,15 @@ Function24423: ; 24423
 	ld a, [VramState]
 	bit 0, a
 	ret z
-	xor a
+	xor a ; effectively ld a, BANK(sScratch)
```

Comment:
Other parts of the code just comment the implied zero value: `xor a ; PARTYMON`, `xor a ; OAKS_POKEMON_TALK`, `xor a ; LOW(vBGMap0)`, etc.